website/docs/guides/duckdb-qs.md

```diff
@@ -0,0 +1,255 @@
+
+In the repository, there's a [`requirements.txt`](https://github.com/dbt-labs/jaffle_shop_duckdb/blob/duckdb/requirements.txt) file which is used to install dbt Core, DuckDB, and all other necessary dependencies. You can check this file to see what will be installed on your machine. It's typically located in the root directory of your project.
+
+The `requirements.txt` file is placed at the top level of your dbt project directory, alongside other key files like `dbt_project.yml`:
+
+
+```shell
+
+/my_dbt_project/
+├── dbt_project.yml
+├── models/
+│   ├── my_model.sql
+├── tests/
+│   ├── my_test.sql
+└── requirements.txt
+
+```
+
+For more information on the setup of DuckDB, you can refer to [DuckDB setup](/docs/core/connect-data-platform/duckdb-setup).
+
+<Tabs>
+  <TabItem value="local" label="Local">
+
+
+1. First, [clone](https://git-scm.com/docs/git-clone) the Jaffle Shop git repository by running the following command in your terminal:
+
+
+
+    ```bash
+    git clone https://github.com/dbt-labs/jaffle_shop_duckdb.git
+
+    ```
+
+2. Change into the docs-duckdb directory from the command line:
+
+    ```shell
+
+    cd jaffle_shop_duck_db
+
+    ```
+
+
+3. Install dbt Core and DuckDB in a virtual environment.
+
+    <Expandable alt_header="Example for Mac" >
+
+    ```shell
+
+    python3 -m venv venv
+    source venv/bin/activate
+    python3 -m pip install --upgrade pip
+    python3 -m pip install -r requirements.txt
+    source venv/bin/activate
+
+    ```
+    </Expandable>
+
+    <Expandable alt_header="Example for Windows" >
+
+    ```shell
+
+    python -m venv venv
+    venv\Scripts\activate.bat
+    python -m pip install --upgrade pip
+    python -m pip install -r requirements.txt
+    venv\Scripts\activate.bat
+
+    ```
+
+    </Expandable>
+
+    <Expandable alt_header="Example for Windows PowerShell" >
+
+    ```shell
+
+    python -m venv venv
+    venv\Scripts\Activate.ps1
+    python -m pip install --upgrade pip
+    python -m pip install -r requirements.txt
+    venv\Scripts\Activate.ps1
+
+    ```
+    </Expandable>
+
+
+4. Ensure your profile is setup correctly from the command line by running the following:
+
+
+    - [dbt compile](https://docs.getdbt.com/reference/commands/compile) &mdash; generates executable SQL from your project source files
+    - [dbt run](https://docs.getdbt.com/reference/commands/run) &mdash; compiles and runs your project
+    - [dbt test](https://docs.getdbt.com/reference/commands/test) &mdash; compiles and tests your project
+    - [dbt build](https://docs.getdbt.com/reference/commands/build) &mdash; compiles, runs, and tests your project
+    - [dbt docs generate](/reference/commands/cmd-docs#dbt-docs-generate) &mdash; generates your project's documentation.
+    - [dbt docs serve](/reference/commands/cmd-docs#dbt-docs-serve) &mdash; starts a webserver on port 8080 to serve your documentation locally and opens the documentation site in your default browser.
+
```

Comment:
might be helpful to add somewhere here that once they do this, they should now have a models folder with staging and marts (i think?) where thy can perform transformations following the version control principles:

<img width="1025" alt="Screenshot 2024-12-23 at 13 00 06" src="https://github.com/user-attachments/assets/4e97723d-9eb8-43da-9b82-a2055c103eaf" />